### ISSUE TITLE
OfflinePlayer api question

### ISSUE THREAD

**Issue by LogGits** [TechFortress/GriefPrevention#756]
So i am trying to use the api to see if an offline player (stored on a sign) has access to the claim that it's in. 
```java
private boolean canAccess () {
    if (GriefPrevention.instance.claimsEnabledForWorld(getMainBlock().getWorld())) {
        Claim claim = GriefPrevention.instance.dataStore.getClaimAt(getMainBlock().getLocation(), true, null);
        if (claim != null) {
            return claim.allowBuild(getOfflinePlayerUUID()) == null; // not a thing
        }
    }
    return false;
}
```
My issue here is that i have a means to get the OfflinePlayer (via the uuid) but i am unsure what the best way to check if the player has access to place blocks is. Is there an efficient way to check if that player is allowed to build in that claim **(that ignores public claims)**.

**Comment by RoboMWM** [comment]
I'm confused. What are you trying to do?

**Comment by LogGits** [comment]
Resolved with #801
